<ui version="4.0" >
 <class>mkLquery</class>
 <widget class="QDialog" name="mkLquery" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>92</x>
     <y>32</y>
     <width>441</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>QSL - DownLoad   &lt; Logbook of The World ></string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>352</y>
     <width>253</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string> 7  Alle bestätigten QSOs</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_1" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>180</y>
     <width>405</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string> 1  Bestätigte QSOs ab   [Last Upload - Datum ]</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>220</y>
     <width>205</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string> 2  QSL  ab [ Datum ]</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>252</y>
     <width>209</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string> 3  QSL  für [ CALL   ]</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>276</y>
     <width>217</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string> 4  QSL  für [ MODE ]</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>300</y>
     <width>217</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string> 5  QSL  für [ BAND  ]</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>324</y>
     <width>217</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string> 6  QSL  für [ DXCC  ]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>68</x>
     <y>132</y>
     <width>309</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Abfrage Queries  ( QSL records )</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonGetRecord" >
   <property name="geometry" >
    <rect>
     <x>68</x>
     <y>424</y>
     <width>217</width>
     <height>32</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>198</red>
         <green>219</green>
         <blue>191</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>198</red>
         <green>219</green>
         <blue>191</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>198</red>
         <green>219</green>
         <blue>191</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string>Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>424</y>
     <width>233</width>
     <height>32</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>197</red>
         <green>196</green>
         <blue>196</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>197</red>
         <green>196</green>
         <blue>196</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>197</red>
         <green>196</green>
         <blue>196</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text" >
    <string>Zurück</string>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>80</y>
     <width>465</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>244</y>
     <width>113</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>216</y>
     <width>133</width>
     <height>33</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>228</red>
         <green>244</green>
         <blue>228</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>228</red>
         <green>244</green>
         <blue>228</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>244</red>
         <green>244</green>
         <blue>244</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="displayFormat" >
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
